ha/note.c

```diff
@@ -169,11 +169,19 @@ static void ha_state_accept(struct m0_confc         *confc,
 	m0_conf_cache_lock(cache);
 	for (i = 0; i < note->nv_nr; ++i) {
 		obj = m0_conf_cache_lookup(cache, &note->nv_note[i].no_id);
+		if (m0_fid_tget(&note->nv_note[i].no_id) == 'r') {
+		M0_LOG(M0_FATAL, "nv_note[%d]=(no_id="FID_F" no_state=%"PRIu32
```

Comment:
why do we want to set log level FATAL

Reply:
Can we have a new trace level for this, something like M0_HA, which has log level between M0_WARN and M0_ERROR? just enough that it gets logged into journal logs/kubectl logs.

Reply:
```
/**
 * Controls which M0_LOG() messages are displayed on console when
 * M0_TRACE_IMMEDIATE_DEBUG is enabled.
 *
 * If log level of trace point is less or equal to m0_trace_level, then it's
 * displayed, otherwise not.
 *
 * By default m0_trace_level is set to M0_WARN. Also, @see documentation of
 * M0_CALL log level, it has special meaning.
 */
extern unsigned int m0_trace_level;
```
```
unsigned int m0_trace_level         = M0_WARN | M0_ERROR | M0_FATAL;
M0_EXPORTED(m0_trace_level);
```
Adding M0_HA to this m0_trace_level should make it print to console.

Reply:
yes, Good option to add new log level.
But Instead of adding M0_HA level,  we should add generic log level which can be use across other module(data-path index path, dtm etc.) whenever we wants to preserve some logs. 
Something like  M0_DEBUG1/M0_DEBUG2